README.md

```diff
@@ -4,14 +4,14 @@ CrowdLeague is a platform for crowd sourcing sports leagues.
 
 Be in a league of your own...
 
-![Github Actions](https://github.com/nickmeinhold/crowdleague/workflows/Github%20Actions/badge.svg)
+![Github Actions](https://github.com/crowdleague/crowdleague/workflows/tests/badge.svg)
 [![Coverage Status](https://coveralls.io/repos/github/crowdleague/crowdleague/badge.svg?branch=dev)](https://coveralls.io/github/crowdleague/crowdleague?branch=dev)
 
 [![GitHub Project](https://img.shields.io/badge/github%20-%23121011.svg?&style=for-the-badge&logo=github&logoColor=white)](https://github.com/crowdleague/crowdleague) 
 [![Firebase Projecct](https://img.shields.io/badge/firebase%20-%23039BE5.svg?&style=for-the-badge&logo=firebase)](https://console.firebase.google.com/u/0/project/crowdleague1/overview)
 
 
-[Google Play Console](https://play.google.com/apps/publish/?account=6095168526928626772#AppDashboardPlace:p=tmp.06172670794154685202.1586081744174.6623537&appid=4973683335528364155) | [App Store Connect](https://appstoreconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/1506440961)
+[Google Play Console](https://play.google.com/console/developers/6095168526928626772/app/4973683335528364155/app-dashboard) | [App Store Connect](https://appstoreconnect.apple.com/WebObjects/iTunesConnect.woa/ra/ng/app/1506440961)
```

Comment:
another sneaky change - just updates the play store listing to the new URL format